api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/GlobalAvgPooling1DTest.kt

```diff
@@ -1,13 +0,0 @@
-package org.jetbrains.kotlinx.dl.api.core.layer
-
-import org.jetbrains.kotlinx.dl.api.core.layer.pooling.GlobalAvgPool1D
-import org.junit.jupiter.api.Test
```

Comment:
agree, that test with zeroes is not the best solution here and in many same cases